File: src/truncate.js
const fs = require('fs');
const content = fs.readFileSync(r'C:\git\cli_ar_homepage\src\app\pages\home\home.component.css', 'utf8');
const lines = content.split('\n');
console.log('Total lines before: ' + lines.length);
const kept = lines.slice(0, 423);
fs.writeFileSync(r'C:\git\cli_ar_homepage\src\app\pages\home\home.component.css', kept.join('\n'), 'utf8');
console.log('Done. File now has ' + kept.length + ' lines.');

File: src/truncate.py
with open(r'C:\git\cli_ar_homepage\src\app\pages\home\home.component.css', 'r', encoding='utf-8') as f:
    lines = f.readlines()
lines = lines[:423]
with open(r'C:\git\cli_ar_homepage\src\app\pages\home\home.component.css', 'w', encoding='utf-8') as f:
    f.writelines(lines)
print(f'Done. File now has {len(lines)} lines.')
